json-lint/schemas/inventory/adservers/a9.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$id": "#a9",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Amazon adserver",
+  "description": "https://github.com/ampproject/amphtml/blob/master/ads/a9.md",
+  "additionalProperties": false,
+  "required": [
```

Comment:
data-adinstanceid ??  
I guess this is totally required, if the goal is to make money 😆  🔢

Reply:
I can put it mandatory, but the documentation only states that one

Reply:
In some examples, we don't have instanceId. However, as talked offline, if we need to remove it, should be quick.